apps/dashboard/blueprints.py

```diff
@@ -10,6 +11,14 @@
 from apps.mapideas import phases as mapideas_phases
 from apps.topicprio import phases as topicprio_phases
 
+# this overwrites adhocracy4.dashboard.blueprints.ProjectBlueprint in order to
+# add the blueprint type
+ProjectBlueprint = namedtuple(
+    'ProjectBlueprint', [
+        'title', 'description', 'content', 'image', 'settings_model', 'type'
+    ]
+)
+
```

Comment:
Not sure about adding it here, as we are eventually also going to add it to other projects? We could also add another ProjectBlueprintWithType or sth to A4?Or ModuleBlueprint :stuck_out_tongue_winking_eye:  @fuzzylogic2000

Reply:
Can we also add it and just not use (leave out) in other projects? Or does it have to have a type as soon as we add it? Or only add the type when the settings are there?
Otherwise, I am also fine with having a ProjectBlueprintWithType in a4. :)

Reply:
:( Well, then I don't really have an opinion. Whatever you think is better is fine with me!

Reply:
Same here :) But then I will leave it here and we can move it once we use it somewhere else..